.github/workflows/system-tests-opensource.yml

```diff
@@ -18,10 +18,17 @@ permissions:
   contents: read
   packages: write
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+  
 on:
    pull_request:
```

Comment:
As i understand, we need this to be on push. 
Otherwise, I don't understand how we expect dev to iterate and fix the failure since the test will run only on the first push (correct me if i'm wrong?)

Reply:
They need to open a PR ( even a draft ) and this will run. We don' t want to run this on every push in early stages to avoid high load on runners. When they need, they open a PR, if they don' t need  to run it they can push without loading the system